interfaces/Portalicious/src/app/pages/project-payment/project-payment.page.ts

```diff
@@ -150,11 +152,11 @@ export class ProjectPaymentPageComponent {
     return new DatePipe(this.locale).transform(date, 'short') ?? '';
   });
 
-  paymentTitle = computed(() => {
-    return $localize`Payment` + ' ' + this.paymentDate();
-  });
+  readonly paymentTitle = computed(
+    () => $localize`Payment` + ' ' + this.paymentDate(),
```

Comment:
Lets fix in: https://dev.azure.com/redcrossnl/121%20Platform/_workitems/edit/33426